.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "jaeger-ui"]
 	path = jaeger-ui
 	url = https://github.com/uber/jaeger-ui
+[submodule "zipkin-api"]
+	path = zipkin-api
+	url = https://github.com/openzipkin/zipkin-api
```

Comment:
What exactly are we getting from zipkin? I would rather do it without submodule dependency

Reply:
If it's just one file we should simply copy it, similar to how we copy the thrift IDL file.

Reply:
It's only one file https://github.com/openzipkin/zipkin-api/blob/master/zipkin2-api.yaml The repo only contains API definitions: thrift and swagger. 

I wanted to propose to use that also for `zipkin.thrift`. Or we can just simply copy swagger to our idl. I don't have a strong preference. Zipkin submodule is easier to maintain. For example our zipkin thrift definition file slightly differs from upstream.

Reply:
+1 to copy

Reply:
https://github.com/jaegertracing/jaeger-idl/pull/39